sphinx/domains/c.py

```diff
@@ -12,7 +12,6 @@
 from sphinx.addnodes import pending_xref
 from sphinx.application import Sphinx
 from sphinx.builders import Builder
-from sphinx.deprecation import RemovedInSphinx60Warning
```

Comment:
For now please bump this to ``RemovedInSphinx70Warning``. We need to confirm with the kernel folks which minimum Sphinx version they use.

Reply:
Are we only keeping this code for the Linux kernel? If so, perhaps a specific opt-in environment variable rather than keeping on moving the depreciation?)

Who would be best to talk to on the kernel side to check if they've updated their syntax?

A

Reply:
@jnikula You mentioned [here](https://github.com/sphinx-doc/sphinx/issues/7008#issuecomment-817309815) that "I did the bulk of the work to convert the Linux kernel to use Sphinx." -- would you be able to comment from the kernel side on what your needs/wants are, or suggest someone else we could ask please?

A

Reply:
Posted at https://lore.kernel.org/linux-doc/LO3P123MB26810D190462B6BBBF1305F6C2A19@LO3P123MB2681.GBRP123.PROD.OUTLOOK.COM/T/#u

A

Reply:
The outcome of that posting was that the kernel doesn't require it. Python still does though, so I'll remove it from this PR.

A